provisioning/service.json5

```diff
@@ -0,0 +1,18 @@
+// Service configuration for production.
```

Comment:
@stegru can you incorporate the service.json5 config file that we're shipping in [morphic-1.3.x](https://github.com/GPII/gpii-app/blob/674dbb2434f8be8dc17b6ef14e3839ef16b3e0bd/provisioning/service.json5)?
Also, extra points if you:
 * update GPII_CLOUD_URL to point to stg rather than prd
 * update autoUpdate.url to point to gpii-app/master/siteconfig.json5

Reply:
done